app/views/main.scala.html

```diff
@@ -208,12 +208,7 @@
         "googleAnalytics" -> googleAnalytics,
         "gaCustomEvent" -> gaCustomEvent,
         "scriptElems" -> Seq(
-            Map("url" -> controllers.routes.AssetsController.versioned("javascript/tai-new.js")),
-            Map("url" -> controllers.routes.AssetsController.versioned("javascript/tai-backlink.js")),
-            Map("url" -> controllers.routes.AssetsController.versioned("javascript/char-count.js")),
-            Map("url" -> controllers.routes.AssetsController.versioned("javascript/card.js")),
-            Map("url" -> controllers.routes.AssetsController.versioned("javascript/tax-code-change.js")),
-            Map("url" -> controllers.routes.AssetsController.versioned("javascript/urbanner.js"))
+            Map("url" -> controllers.routes.AssetsController.versioned("javascripts/tai-app.js"))
```

Comment:
same change(s) required in this main to the "other" main views e.g. `https://github.com/hmrc/tai-frontend/blob/master/app/views/main_without_nav.scala.html`

Reply:
done